eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/ISBNConverter.php

```diff
@@ -82,7 +82,7 @@ public function toFieldDefinition(StorageFieldDefinition $storageDef, FieldDefin
     {
         $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
             array(
-                'isISBN13' => !empty($storageDef->dataInt1) ? (bool)$storageDef->dataInt1 : true,
+                'isISBN13' => !empty($storageDef->dataInt1),
```

Comment:
heh, that's (almost) exactly how it was done before https://github.com/ezsystems/ezpublish-kernel/pull/1983/commits/f320360bd8e12f3b96b55d519ee2cd662430e8dd - I changed it according to @alongosz suggestions ;)